model/authentication.go

```diff
@@ -73,6 +73,17 @@ func (auth *Authentication) ToResponse() *AuthenticationResponse {
 	}
 }
 
+func (auth *Authentication) SecretToResponse() *SecretResponse {
```

Comment:
What would you think about `ToSecretResponse()` to follow our other response names?